test/lit.cfg

```diff
@@ -376,6 +376,14 @@ if run_cpu == "i386" or run_cpu == "x86_64":
 if "optimized_stdlib" in config.available_features:
   config.available_features.add("optimized_stdlib_" + run_cpu)
 
+# Check to see if there's a compiler_rt directory.
+# FIXME: We should really plumb this through build-script and CMake instead of
```

Comment:
This is fine for a first pass, but please send a radar over to me so we don't forget this.